pkg/render/applicationlayer/applicationlayer.go

```diff
@@ -397,14 +433,16 @@ func (c *component) volumes() []corev1.Volume {
 		volumes = append(volumes, corev1.Volume{
 			Name: DikastesSyncVolumeName,
 			VolumeSource: corev1.VolumeSource{
-				EmptyDir: &corev1.EmptyDirVolumeSource{},
+				HostPath: &corev1.HostPathVolumeSource{
+					Path: "/var/run/dikastes",
+					Type: &hostPathDirectoryOrCreate,
+				},
 			},
 		})
 
 		// Needed for ModSecurity library - contains rule set.
-		if c.config.WAFEnabled { // WAF-only
+		if c.config.WAFEnabled || c.config.SidecarInjectionEnabled {
 			// WAF logs need HostPath volume - logs to be consumed by fluentd.
```

Comment:
Does this mean that the per-host daemonset is still responsible for L7 logging even if the sidecar mode is being used?

Reply:
Yes... actually the per-host daemonset is still doing everything for l7 features in EE, on sidecars, only the envoy proxy is injected on the application pods.